lib/Chat/Changelog/Manager.php

```diff
@@ -69,64 +69,68 @@ public function updateChangelog(string $userId): void {
 	public function getChangelogs(): array {
 		return [
 			$this->l->t(
-				"Welcome to Nextcloud Talk!\n"
+				"# Welcome to Nextcloud Talk!\n"
```

Comment:
Not sure we should use `<h1>`, as per CEO / readers / someone else's standards it's required to use no more than 1 header on a web-page

Maybe stick with `##` for big headers, and `###` for subheaders?

Reply:
```suggestion
				"## Welcome to Nextcloud Talk!\n"
```